docs/development/how-to/ide-setup/eclipse.md

```diff
@@ -23,8 +23,7 @@
 
 ## Formatter
 1. [Install Google Java Format Plugin](https://github.com/google/google-java-format#eclipse)
-   * ***Important*** Use the unofficial 1.7 version available here: https://github.com/google/google-java-format/files/2774507/google-java-format-eclipse-plugin-1.7.0.jar.zip
-   * Download the 1.7 version, unzip it, and move the extracted 'jar' file to the eclipse 'dropins' folder.
+   * Download the most recent version, unzip it, and move the extracted 'jar' file to the eclipse 'dropins' folder.
```

Comment:
Is this comment still necessary? It looks like anyone following the link would read the same information: https://github.com/google/google-java-format#eclipse

FWIW, to explain my thinking here: for documentation, I certainly believe 'less is more'. More documentation is more things to maintain, and more stuff for a new person to sift through to find the really useful information (and so lots of documentation lowers the value of the useful documentation, makes it harder to find).

Reply:
In general I agree that 'less is more', but having multiple links to open and keeping an overview is making it more cumbersome. It is more user-friendly (especially for new users) to have one complete guide on a page that they can follow. 
Even if this guide will be incorrect over time, the user still can make/request updates (same as for broken links) or can try to figure it out for himself with the link provided.

Reply:
> but having multiple links to open and keeping an overview is making it more cumbersome.

How do you mean by this? Which multiple links exactly?